### PR TITLE
[Update] Install self-hosted TimescaleDB from a pre-built cloud image

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Created a new TimescaleDB instance on AWS EC2 and updated the instructions.

The commands `psql -U postgres -h localhost` are applicable only if you installed in on self hosted local host.
# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]
Screenshot 2022-02-23 at 11 47 40 AM

Each region now contains multiple AMIs. for example us-east-1 has multiple AMIs.Screenshot 2022-02-24 at 8 04 16 AM

### CODE REVIEW

File: install/installation-cloud-image.md [hunk 1]
```diff
@@ -49,80 +49,53 @@ supports public AMIs.
...
 </procedure>
 
-When you have completed the installation, you need to configure your database so
-that you can use it. The easiest way to do this is to run the `timescaledb-tune`
-script, which is included with the `timescaledb-tools` package. For more
-information, see the [configuration][config] section.
+After you have completed the installation, connect to your instance and configure your database. For information about connecting to the instance, see the AWS [accessing instance documentation][aws-connect]. The easiest way to configure your datase is to run the `timescaledb-tune` script, which is included with the `timescaledb-tools` package. For more information, see the [configuration][config] section. 
+
+<highlight type="note">
+After you run the *timescaledb-tune* script you need to restart the PostgreSQL service for the configuration changes to take effect. To restart the service use the **sudo systemctl restart postgresql.service** command.
```
Comment (charislam) [inline install/installation-cloud-image.md:55]:
```suggestion
After running the `timescaledb-tune` script, you need to restart the PostgreSQL service for the configuration changes to take effect. To restart the service, run `sudo systemctl restart postgresql.service`.
```

File: install/installation-cloud-image.md [hunk 2]
```diff
@@ -49,80 +49,53 @@ supports public AMIs.
...
+</highlight>
+
 
 ## Set up the TimescaleDB extension
-When you have PostgreSQL and TimescaleDB installed, you can connect to it from
-your local system using the `psql` command-line utility. This is the same tool
-you might have used to connect to PostgreSQL before, but if you haven't
-installed it yet, check out our [installing psql][install-psql] section.
+When you have PostgreSQL and TimescaleDB installed, connect to your instance and setup the TimescaleDB extension.
```
Comment (charislam) [inline install/installation-cloud-image.md:60]:
```suggestion
When you have PostgreSQL and TimescaleDB installed, connect to your instance and set up the TimescaleDB extension.
```

File: install/installation-cloud-image.md [hunk 3]
```diff
@@ -49,80 +49,53 @@ supports public AMIs.
...
-    ```
-    psql (13.3, server 12.8 (Ubuntu 12.8-1.pgdg21.04+1))
-    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
-    Type "help" for help.
-    tsdb=>
+    sudo -u postgres psql
     ```
-1.  At the `psql` prompt, create an empty database. Our database is
-    called `example`:
+1.  At the prompt, create an empty database. Our database is
```
Comment (charislam) [inline install/installation-cloud-image.md:70]:
```suggestion
1.  At the prompt, create an empty database. For example, to create a database
```

**Review by charislam — APPROVED** [review]

File: install/installation-cloud-image.md [hunk 4]
```diff
@@ -40,89 +25,61 @@ supports public AMIs.
 ### Installing self-hosted TimescaleDB from a pre-build cloud image
 1.  Make sure you have an [Amazon Web Services account][aws-signup], and
     are signed in to [your EC2 dashboard][aws-dashboard].
-1.  Navigate to `Images → AMIs`.
-1.  In the search bar, change the search to `Public images` and enter the
-    Image ID for your region. Alternatively, you can use the `Timescale` search
+1.  Navigate to **Images > AMIs**.
```
Comment (Loquacity) [inline install/installation-cloud-image.md:28]:
```suggestion
1.  Navigate to `Images → AMIs`.
```

File: install/installation-cloud-image.md [hunk 5]
```diff
@@ -40,89 +25,61 @@ supports public AMIs.
...
+1.  In the search bar, change the search to `Public images` and type *Timescale* search
     term to find all available Timescale images.
-1.  Select the image you want to use, and click `Launch`.
+1.  Select the image you want to use, and click **Launch instance from image**.
```
Comment (Loquacity) [inline install/installation-cloud-image.md:31]:
```suggestion
1.  Select the image you want to use, and click `Launch instance from image`.
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Comment by Rajakavitha1** [comment]
Hi @Loquacity !!! Just incorporated the changes that you suggested.

**Review by Loquacity — APPROVED** [review]

**Comment by svenklemm** [comment]
What is the difference between install/installation-cloud-image.md and install/installation-ubuntu-ami.md? Can we drop the latter?

**Comment by Rajakavitha1** [comment]
> What is the difference between install/installation-cloud-image.md and install/installation-ubuntu-ami.md? Can we drop the latter?

@Loquacity  can you please help me answer this for @svenklemm.  I did read the contents of install/installation-ubuntu-ami.md and yes, may be we can drop that.

**Comment by Loquacity** [comment]
> > What is the difference between install/installation-cloud-image.md and install/installation-ubuntu-ami.md? Can we drop the latter?
> 
> @Loquacity can you please help me answer this for @svenklemm. I did read the contents of install/installation-ubuntu-ami.md and yes, may be we can drop that in a different PR. @svenklemm could you please review this PR for technical accuracy and state your approval.

install/installation-ubuntu-ami.md is not published. It can be deleted.

**Comment by Rajakavitha1** [comment]
> > > What is the difference between install/installation-cloud-image.md and install/installation-ubuntu-ami.md? Can we drop the latter?
> > 
> > 
> > @Loquacity can you please help me answer this for @svenklemm. I did read the contents of install/installation-ubuntu-ami.md and yes, may be we can drop that in a different PR. @svenklemm could you please review this PR for technical accuracy and state your approval.
> 
> install/installation-ubuntu-ami.md is not published. It can be deleted.

Deleted the file install/installation-ubuntu-ami.md based on the review feedback

**Review by svenklemm — APPROVED** [review]
LGTM, i like that we no longer have to update the image ids going forward.

File: install/installation-cloud-image.md [hunk 6]
```diff
@@ -41,88 +26,60 @@ supports public AMIs.
...
+1.  Select the image you want to use, and click `Launch instance from image`.
     <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/aws_launch_ami.png" alt="Launch an AMI in AWS EC2"/>
 
 </procedure>
 
-When you have completed the installation, you need to configure your database so
-that you can use it. The easiest way to do this is to run the `timescaledb-tune`
-script, which is included with the `timescaledb-tools` package. For more
-information, see the [configuration][config] section.
+After you have completed the installation, connect to your instance and configure your database. For information about connecting to the instance, see the AWS [accessing instance documentation][aws-connect]. The easiest way to configure your datase is to run the `timescaledb-tune` script, which is included with the `timescaledb-tools` package. For more information, see the [configuration][config] section.
```
Comment (Loquacity) [inline install/installation-cloud-image.md:36]: Break at 80 chars

**Review by Loquacity — APPROVED** [review]
Approved, but please fix the line breaks before merge.

File: install/installation-cloud-image.md (hunk 6 above)
Comment (Rajakavitha1) [reply install/installation-cloud-image.md]: added line breaks

**Pull request merged**